src/fr/mrc/ptichat/main/Main.java

```diff
@@ -1,11 +1,34 @@
 package fr.mrc.ptichat.main;
 
+import fr.mrc.ptichat.connection.Client;
+import fr.mrc.ptichat.connection.ClientServer;
+import fr.mrc.ptichat.connection.Input;
+
+import java.util.Scanner;
+
 /**
  * Created by roxane on 11/10/2016.
  */
 public class Main {
 
-    public boolean testExample() {
-        return true;
+    /**
+     * Creates the chat in the console
+     * @param args
+     */
+    public static void main(String[] args) {
+        // TODO: Replace this method by linking the classes ClientServer and Client to the UI (JFrame)
```

Comment:
To keep our ```main``` functions (while all is not connected) maybe we could do something like that :
```java
public class Main {
    static void uiProgramm() {
         //insert roxane's main code here
    }

    static void connectionProgramm() {
         //insert melanie's main code here
    }

    public static void main(String[] args) {
        Main.connectionProgramm();
    }
}
```